Rex/Misc/Sparrow/__module__.pm

```diff
@@ -0,0 +1,142 @@
+package Rex::Misc::Sparrow;
+
+use Rex -base => [qw(1.4 exec_autodie)];
+use Rex::CMDB;
+
+use Rex::Lang::Perl::Cpanm;
+
+use File::Spec;
+use YAML;
+
+our $sparrow = get cmdb 'sparrow';
```

Comment:
Not sure, but looks like `get cmdb 'sparrow'` always return undef , see my comments at pull request ...